src/mathematics/glyphinfo.rs

```diff
@@ -0,0 +1,142 @@
+use truetype::{Result, Tape, Value};
+
+use layout::Coverage;
+use super::ValueRecord;
+
+table! {
+    @position
+    #[doc = "A table of positioning information defined on a per-glyph bases."]
+    pub GlyphInfo { // MathGlyphInfo
```

Comment:
I propose renaming it to `Glyphs`, analogous to `Constants` and `Variants`. There’ll be no confusion about what kind of glyphs we’re talking about there; we’re in the `mathematics` module.